Bottom: 525fc9a1a5a941989abeaf096b23523314ccc0e4
Top:    525fc9a1a5a941989abeaf096b23523314ccc0e4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-04 22:56:59 +0000

Try adding polygon objects after each clipping stage. rtree->faster??

*** ACTUALLY SLOWS STARTUP DOWN ***







---


